nuxeo-drive-client/nxdrive/engine/watcher/local_watcher.py

```diff
@@ -611,11 +611,6 @@ def _handle_watchdog_event_on_known_pair(self, doc_pair, evt, rel_path):
             if is_office_temp_file(dest_filename):
                 log.debug('Ignoring Office tmp file: %r', evt.dest_path)
                 return
-            # Ignore normalization of the filename on the file system
-            # See https://jira.nuxeo.com/browse/NXDRIVE-188
-            if evt.dest_path == normalize_event_filename(evt.src_path):
-                log.debug('Ignoring move from %r to normalized name: %r', evt.src_path, evt.dest_path)
-                return
```

Comment:
The old way was a copy paste in the two methods instead of handling correctly before even getting to those methods